content/docs/developer-resources/forms/index.md

```diff
@@ -48,7 +48,7 @@ export class FormsPage {
 }
 ```
 
-This is very similar to how forms were made in Angular 1/Ionic 1 and will be the most familiar to other developers.
+This is very similar to how forms were made in Angular 1/Ionic V1 and will be the most familiar to other developers.
```

Comment:
I don't know if this change is necessary since Angular is still written here as Angular 1